core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.spec.ts

```diff
@@ -1472,6 +1528,21 @@ describe('State translation component', () => {
     expect(htmlData).toBe('<p>HTML data</p>');
   });
 
+  it('should return translation unicode when translation no available', () => {
```

Comment:
I don't understand what's going on in this test. It doesn't seem to be testing what it's supposed to.

Reply:
@seanlip i have addressed all the comments.